### ISSUE TITLE
Should we have to define "EPUB Publication Instance"?

### ISSUE THREAD

**Issue by iherman** [mattgarrish/epub-revision#1932]
(This is an [editors' note in the spec](https://w3c.github.io/epub-specs/epub33/rs/#sec-container-iri) converted into a github issue.)

The [definition for URL's of the Root directory](https://w3c.github.io/epub-specs/epub33/rs/#sec-container-iri) says:

> The origin of the container root URL is unique for each EPUB Publication instance.

Do we have to define what "EPUB Publication instance" is? Or should we consider as some sort of common knowledge?

**Comment by mattgarrish** [comment]
If the text elaborating on what it means weren't in a note, it probably would be fine to omit any further explanation.

Maybe just strip the note formatting?

**Comment by iherman** [comment]
> If the text elaborating on what it means weren't in a note, it probably would be fine to omit any further explanation.
> 

My problem is not that it is in a note; the term  appears twice in normative texts: [§4.4](https://w3c.github.io/epub-specs/epub33/rs/#sec-container-iri) and [§7.1.1](https://w3c.github.io/epub-specs/epub33/rs/#sec-container-iri). If you think that it is fine to omit any further explanation, then the third occurrence in the note does not change anything...

The question is whether it does require further explanation or not. (Personally, I am fine leaving the text without it.)

**Comment by mattgarrish** [comment]
> the term appears twice in normative texts

But we can solve that with a dfn tag on the first instance. We don't have to go to the terminology section for a technical definition like this, especially when it has limited application. I'd keep the terminology section for terms that are widely used throughout the specifications.

But I don't like having to read a note to find out what an "instance" is. It should be clearer in the normative text, otherwise it is easily confused with being nothing more than a ten cent way of saying "an EPUB Publication".

I wonder if there's some other way of phrasing it in the third bullet, though, to make its meaning more apparent. This is maybe a bit wordy, but something like:

> The origin of the container root URL is unique for each user-specific instance of an EPUB Publication in a Reading System.

I could live with the elaboration in a note, then, as it would follow more directly from the bullet.

**Comment by iherman** [comment]
I sort of like "user-specific instance of ab EPUB Publication" and I would be o.k. exchanging all occurrences in §4.4 and §7.1.1. It is a bit verbose, but it makes the intention right and leaves it clearer that, in some sense, this "instance" is very much dependent on the particular RS.

It, however, suggests some situations that, actually, may be right. If I have a book in apple's ibooks, I can read the same book from my iPad and my iPhone, the two reading cases share, for example, bookmarks. This suggests that those are one user specific instance, ie, the origin may be identical. This sounds actually all right to me...

@bduga and @rdeltour, as partners in crime behind this issue, wdyt?

**Comment by rdeltour** [comment]
No strong opinion here. If we don't want to further specify what "instance" means, I agree that the suggested addition of "user-specific instance in a Reading System" sounds like an improvement over simply "instance".

**Comment by iherman** [comment]
Ok, created #1928 to possibly close this issue.
